epicure-frontend/src/components/CardSwipper/CardSwipper.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../styles.scss";
+
+.swiper-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 19px;
```

Comment:
usually it's not very good if you have margin only on one side like this
it may cause issues on the other side in responsive modes